<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window</class>
 <widget class="QMainWindow" name="window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>411</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>372</width>
    <height>411</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>372</width>
    <height>411</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>eyepassgen - Password Generator (AHXR)</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(20, 20, 20);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frTop">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>16777215</width>
      <height>16777215</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lblLength">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>eyepassgen</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/icon.jpg);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>50</y>
      <width>391</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/bg.jpg) 0 0 0 0 stretch stretch;
background-repeat: no-repeat;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>351</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="title">
      <string>Settings</string>
     </property>
     <widget class="QCheckBox" name="chkNumbers">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>65</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Gothic</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);
background-image: url(:/1.png);</string>
      </property>
      <property name="text">
       <string>Numbers</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkLowercase">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>77</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Gothic</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);
background-image: url(:/1.png);</string>
      </property>
      <property name="text">
       <string>Lowercase</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkUppercase">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>77</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Gothic</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);
background-image: url(:/1.png);</string>
      </property>
      <property name="text">
       <string>Uppercase</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkStarting">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Gothic</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);
background-image: url(:/1.png);</string>
      </property>
      <property name="text">
       <string>Starting Letter</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkSymbols">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>77</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Gothic</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);
background-image: url(:/1.png);</string>
      </property>
      <property name="text">
       <string>Symbols</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkUCharacters">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Gothic</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);
background-image: url(:/1.png);</string>
      </property>
      <property name="text">
       <string>Unique Characters</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkUSequence">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>60</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Gothic</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);
background-image: url(:/1.png);</string>
      </property>
      <property name="text">
       <string>Unique Sequence</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbLength">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>80</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </widget>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>351</width>
       <height>19</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);
background-image: url(:/1.png);</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>150</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btnGenerate">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>351</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Generate Password</string>
     </property>
    </widget>
    <widget class="QListWidget" name="lstPasswords">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>351</width>
       <height>141</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Gothic</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);
background-image: url(:/1.png);</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <signal>mousePressEvent(QMouseEvent*event)</signal>
 </slots>
</ui>
